frontend/packages/knative-plugin/src/components/overview/__tests__/RevisionsOverviewList.spec.tsx

```diff
@@ -81,7 +113,47 @@ describe('RevisionsOverviewList', () => {
     expect(wrapper.find(Button).exists()).toBe(false);
   });
 
-  it('should render RevisionsOverviewListItem', () => {
-    expect(wrapper.find(RevisionsOverviewListItem)).toHaveLength(1);
+  it('should render RevisionsOverviewListItem for revisions as many as MAX_REVISION if number of revisions receiving traffic is less than MAX_REVISION', () => {
+    wrapper = shallow(
+      <RevisionsOverviewList
+        revisions={exceedMaxRevisions}
+        service={MockKnativeResources.revisions.data[0]}
+      />,
+    );
+    expect(wrapper.find(RevisionsOverviewListItem)).toHaveLength(3);
+  });
+
+  it('should render RevisionsOverviewListItem for all revisions receiving traffic', () => {
+    const traffic = [
+      {
+        latestRevision: false,
+        percent: 25,
+        revisionName: 'overlayimage-tkvz5',
+      },
+      {
+        latestRevision: false,
+        percent: 25,
+        revisionName: 'overlayimage-bwpxq',
+      },
+      {
+        latestRevision: false,
+        percent: 25,
+        revisionName: 'overlayimage-n2b7n',
+      },
+      {
+        latestRevision: false,
+        percent: 25,
+        revisionName: 'overlayimage-m3x8m',
+      },
+    ];
```

Comment:
can we move this to mock-data file?

Reply:
use existing `traffic-split-utils-mock.ts`

Reply:
done